llvm/tools/llvm-mca/Views/InstructionInfoView.h

```diff
@@ -65,28 +66,39 @@ class InstructionInfoView : public InstructionView {
 
   struct InstructionInfoViewData {
     unsigned NumMicroOpcodes = 0;
+    // Latency + ForwardingDelayCycles: negative ReadAdvance
     unsigned Latency = 0;
+    // ReadAvance Bypasses cycles: Latency - ReadAdvance (positive value)
+    unsigned Bypass = 0;
     std::optional<double> RThroughput = 0.0;
     bool mayLoad = false;
     bool mayStore = false;
     bool hasUnmodeledSideEffects = false;
+    std::string OpcodeName = "";
```

Comment:
judging from your uses, this is storing opcode names returned from `MCInstrInfo::getName`: that function gives you a string from a giant `static` table, which is why it returns a `StringRef`. So I think you can use `StringRef` to avoid extra allocation induced by `std::string`.